### PR TITLE
Add search to pkg inspector

### LINKED ISSUE #1196: Request to add search functionality to Package Inspector App
Can we add search functionality to package inspector app?
 
If there are a lot of packages on the device, it gets hard to find the package I'm looking for. A search bar at the top would be nice to have

### PR DESCRIPTION
**Pull request by iambmelt**
Closes #1196

### CODE REVIEW

File: pop-benchmarker/src/main/java/com/microsoft/identity/common/internal/platform/MainActivity.java [hunk 1]
```diff
@@ -251,7 +251,7 @@ private void getIsHardwareIsolated(@NonNull final AsyncResultCallback<String> ca
             final Timer.TimerResult<Boolean> result = Timer.execute(new Callable<Boolean>() {
                 @Override
                 public Boolean call() throws Exception {
-                    return IDevicePopManager.SecureHardwareState.TRUE_UNATTESTED
+                    return SecureHardwareState.TRUE_UNATTESTED
```
Comment (iambmelt) [inline pop-benchmarker/src/main/java/com/microsoft/identity/common/internal/platform/MainActivity.java:254]: Not sure what this is doing here.... I'll remove from this PR

Comment (iambmelt) [reply pop-benchmarker/src/main/java/com/microsoft/identity/common/internal/platform/MainActivity.java:254]: Ah -- this is related to the submodule. Will push an update to it.

File: package-inspector/src/main/res/menu/options_menu.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@+id/search"
+        android:title="Search Pkgs"
+        android:icon="@android:drawable/ic_search_category_default"
+        app:showAsAction="collapseActionView|ifRoom"
+        app:actionViewClass="android.widget.SearchView" />
+</menu>
```
Comment (shahzaibj) [inline package-inspector/src/main/res/menu/options_menu.xml:9]: new line at end of file

Comment (iambmelt) [reply package-inspector/src/main/res/menu/options_menu.xml]: d075004e99b5b1eede33944d1832c3f8015e420c

**Review by shahzaibj — APPROVED** [review]

**Comment by shahzaibj** [comment]
Thanks @iambmelt

**Pull request merged**